task-list/app/views/tasks/index.html.erb

```diff
@@ -3,28 +3,37 @@
   <ul>
     <% @tasks.each do |task| %>
       <% if task.completed_date == nil %>
-        <% @completion_class = "incomplete_button" %>
+        <% @completion_id = "incomplete_button" %>
       <% else %>
-        <% @completion_class = "complete_button" %>
+        <% @completion_id = "complete_button" %>
       <% end %>
-      <li>
-        <%= button_to "✓","/tasks/#{task.id}/complete", method: :patch, class: @completion_class %>
-        <a href= "/tasks/<%= task.id %>"><%= task.name %></a>
+      <li class="task_list", id="holder">
```

Comment:
Because this `li` tag and the `div` tags within it are all within a loop, they will potentially be repeated in the output HTML. For that reason we should not use `id` attributes without making them unique by doing something like `id="holder_<%= task.id %>"`, or by writing a view helper method to do the same thing.

In this case it looks like the `id` values could probably be considered classes, so we could change them to classes instead by moving them into the `class` attribute: `<li class="task_list holder">`.